openwisp-config/files/openwisp.agent

```diff
@@ -508,6 +508,16 @@ fix_uci_config() {
 		       -t openwisp \
 		       -p daemon.info
 	fi
+
+	# Fix md5sum for service reload
```

Comment:
For some reason I cannot edit this file.

I would like to make this comment more explicit, eg:

```
Regenerate the md5 checksums to avoid reloading services which had anonymous UCI sections which were removed.
```

Or something like that.

Reply:
OK, I will update the comment.